package.json

```diff
@@ -12,23 +12,25 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "autotrack": "^1.0.0",
-    "babel-core": "^6.11.4",
-    "babel-eslint": "^6.1.2",
-    "babel-loader": "^6.2.4",
-    "babel-polyfill": "^6.9.1",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-0": "^6.5.0",
-    "camelcase": "^3.0.0",
-    "core-js": "^2.4.1",
+    "babel-core": "^6.1.13",
+    "babel-eslint": "^6.0.2",
+    "babel-loader": "^6.2.0",
+    "babel-polyfill": "^6.3.13",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "camelcase": "^2.0.1",
+    "chart.js": "^1.1.1",
+    "core-js": "^2.0.2",
     "create-output-stream": "0.0.1",
-    "del": "^2.2.1",
-    "eslint": "^3.1.1",
-    "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-react": "^5.2.2",
-    "fs-extra": "^0.30.0",
-    "glob": "^7.0.5",
-    "gulp": "^3.9.1",
+    "del": "^2.2.0",
+    "eslint": "^2.8.0",
+    "eslint-plugin-babel": "^3.0.0",
+    "eslint-plugin-react": "^4.3.0",
+    "fixed-data-table-2": "^0.7.1",
+    "fs-extra": "^0.27.0",
+    "glob": "^7.0.3",
+    "gulp": "^3.9.0",
     "gulp-autoprefixer": "3.1.0",
     "gulp-concat": "^2.6.0",
     "gulp-cssnext": "^1.0.1",
```

Comment:
I think `moment` is missing here. I got a build error when trying to run the site. If you delete your `node_modules` folder and run `npm install` again, you'll be able to see if anything's missing.